mbedtls-sys/vendor/ChangeLog.d/muladdc-amd64-memory.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
```

Comment:
:wrench: It looks like this got removed in https://github.com/Mbed-TLS/mbedtls/pull/4978 and was consolidated to `muladdc-memory.txt`

Reply:
good catch